pom.xml

```diff
@@ -551,7 +557,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-release-plugin</artifactId>
-        <version>3.0.0-M6</version>
+        <version>3.0.1</version>
```

Comment:
propose to update other plugin versions as well, you can use the same as here for example:
https://github.com/folio-org/mod-orders/pull/851/files

Reply:
Done. Thank you